services/static-webserver/client/source/class/osparc/dashboard/WorkspacesAndFoldersTree.js

```diff
@@ -228,6 +228,9 @@ qx.Class.define("osparc.dashboard.WorkspacesAndFoldersTree", {
 
       const sharedWorkspaceModel = this.__getModel(-1, null);
       sharedWorkspaceModel.getChildren().append(workspaceModel);
+      sharedWorkspaceModel.getChildren().sort(((a, b) => {
```

Comment:
Just a small question, I assume this is case-aware. I wonder if it maybe shouldn't be?

Reply:
agreed. Don't we have a utils function for sorting?

Reply:
By default it isn't
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator#sensitivity